app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryUpdate.kt

```diff
@@ -31,6 +31,12 @@ class CustomDrugEntryUpdate : Update<CustomDrugEntryModel, CustomDrugEntryEvent,
     }
   }
 
+  private fun frequencyEdited(nextModel: CustomDrugEntryModel, event: FrequencyEdited): Next<CustomDrugEntryModel, CustomDrugEntryEffect> {
+    val frequency = if (nextModel.isDrugFrequencyUnknown) null else event.frequency
+
+    return next(nextModel, SetDrugFrequency(frequency), SetSheetTitle(nextModel.drugName, nextModel.dosage, frequency))
```

Comment:
Correct me if I am wrong, instead of triggering UI actions for this, can't we just render it in the UI renderer? Is there any reason we are not doing that?